compat/media/media_buffer_layer.cpp

```diff
@@ -260,12 +299,26 @@ MediaBufferWrapper* media_abuffer_get_media_buffer_base(MediaABufferWrapper *buf
 
 #if ANDROID_VERSION_MAJOR>= 5
 
+#if ANDROID_VERSION_MAJOR>=8
+    android::MediaBufferBase *mbufb = NULL;
+    
+    android::sp<android::RefBase> holder;
+    if (d->buffer->meta()->findObject("mediaBufferHolder", &holder)) {
+        mbufb = (holder != nullptr) ?
+        static_cast<android::MediaBufferHolder*>(holder.get())->mediaBuffer() : nullptr;
```

Comment:
Add some more indentation (for example 4 spaces) to this line so it's clearly continuing the code of the previous line.